app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt

```diff
@@ -133,30 +155,78 @@ class OptionsFragmentPresenter @Inject constructor(
         profileManagementController.updateReadingTextSize(
           profileId,
           ReadingTextSize.SMALL_TEXT_SIZE
+        ).observe(
+          fragment,
+          Observer {
+            if (it.isSuccess()) {
+              readingTextSize = ReadingTextSize.SMALL_TEXT_SIZE
+            } else {
+              consoleLogger.e(
+                READING_TEXT_SIZE_TAG,
+                READING_TEXT_SIZE_ERROR,
```

Comment:
You cannot have the same logger everywhere, the error should be different for each logger otherwise if anyone faces this error how would they know which error is related to which function?

              consoleLogger.e(
                READING_TEXT_SIZE_TAG,
                READING_TEXT_SIZE_ERROR + ": small text size",
                it.getErrorOrNull()
              )

Reply:
Done